proto/destination.proto

```diff
@@ -59,6 +59,13 @@ service Destination {
 message GetDestination {
   string scheme = 1;
   string path = 2;
+  // This identifies the proxy which is requesting the profile.  The server may
+  // use this as context when deciding which profile to return.  For example,
+  // the server may look for the profile in the client's namespace.
+  // For proxies running in Kubernetes, the format of this id should be
+  // "<pod>.<namespace>".
+  string client_id = 3;
```

Comment:
Okay, last nit: can we call this proxy_id? We don't talk about "clients" anywhere. This is all proxy-api. Let's talk about proxies.